src/catala/literate/literate_common.mli

```diff
@@ -0,0 +1,30 @@
+(* This file is part of the Catala compiler, a specification language for tax and social benefits
+   computation rules. Copyright (C) 2020 Inria, contributor: Denis Merigoux
```

Comment:
Please replace my name with yours, you have written this :)

Reply:
For contributions, do I add my name in all the files I have modified or only the ones I have created?

Reply:
Both :)